SM-3942: Update API documentation with Data Export API details. (#110)
departmentfortransport/street-manager-docs: docs/api-documentation/V1.10/index.md

Accepts the user's email address, new password and token (returned from the Work API <code>/authenticate/initial</code> endpoint). This will overwrite the temporary password received by email (from the <code>/invite-user</code> endpoint) and activate the user's account. The response will include the same authentication tokens returned from the Work API <code>/authenticate</code> endpoint.
{: .govuk-body}

### Data Export API
{: .govuk-heading-m}

#### Get latest Work Data CSV
{: .govuk-heading-s}

<code>GET /work-data</code>

> Retrieves data of permits which have been added or changed within the last hour in CSV format. See the Data Export API and Open Data in the Technical Overview for details.

Reviewer: ```suggestion
Retrieves data of permits which have been added or changed within the last hour in CSV format. See the Data Export API and Open Data in the Technical Overview section for details.
```

Version 1.10 (28/11/2019):
{: .govuk-heading-s #upcoming-changes}

<ol class="govuk-list govuk-list--bullet">

>   <li>Now that the Data Export API is available, details on who can access this data, and what this data contains, has been added. Add <code>GET /work-data</code> endpoint. (See the resource guide for details)</li>

Reviewer: ```suggestion
  <li>Add new Data Export API with endpoint <code>GET /work-data</code> endpoint. (See the resource guide for details)</li>
```
